web/src/locales/en.json

```diff
@@ -45,6 +45,13 @@
     "clipboard-error": "Error copying URL to clipboard",
     "share-error": "Error sharing URL"
   },
+  "more-options-dropdown": {
+    "title": "Share",
+    "mobile-share-via": "Share via",
+    "copy-chart-link": "Copy link to chart",
+    "preliminary-data": "Preliminary data may change over time.",
+    "summary": "Discover real-time electricity insights with the @electricitymaps app!"
```

Comment:
I raised this in Figma as well but having @ in here will only work on specifik platforms such as twitter. But not on facebook (I think), reddit, email etc.

I think it would be best to remove it if we can't make it platform specific.

Reply:
Sure, how about switching it from `...the @electricitymaps app!` -> `...the Electricity Maps app!`?

Reply:
That works!